mmvb_backend/benchmarking_sessions/migrations/0001_initial.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.0.4 on 2020-05-04 13:38
+
+import uuid
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import django_mysql.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ("ai_implementations", "0001_initial"),
+        ("cases", "0002_remove_case_name"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="BenchmarkingSession",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("created_on", models.DateTimeField(auto_now_add=True)),
+                ("modified_on", models.DateTimeField(auto_now=True)),
+                (
+                    "status",
+                    models.CharField(
+                        choices=[
+                            ("created", "Created"),
```

Comment:
`intermediate` was not generated as a choice on the migrations? 🤔 
Maybe it was added later and another migration is missing?

Reply:
🤦